packages/story-editor/src/components/library/panes/text/fontPreview.js

```diff
@@ -161,8 +176,30 @@ function FontPreview({ title, element, insertPreset, getPosition, index }) {
     pageCanvasData,
     versionNumber,
   ]);
-
   const onClick = useCallback(() => {
+    // If we have only text(s) selected, we apply the preset instead of inserting.
+    if (isText) {
+      updateSelectedElements({
```

Comment:
Why is the element being moved as well when applying a text preset? Should the element stay in place?

![preset-move](https://user-images.githubusercontent.com/637548/156235903-a19a9312-ef9c-4486-a474-33c0bcea858a.gif)

And should things like italic, color, underline, all caps, and text alignment also be "reset" rather than just font family, font size, and font weight?